packages/data-stores/src/launch/reducer.ts

```diff
@@ -105,6 +105,30 @@ const isSiteTitleStepVisible: Reducer< boolean, LaunchAction > = ( state = false
 	return state;
 };
 
+const isModalDismissible: Reducer< boolean, LaunchAction > = ( state = true, action ) => {
+	if ( action.type === 'SET_MODAL_DISMISSIBLE' ) {
```

Comment:
Do we think these action type strings would be better assigned to const variables and imported into `actions.ts` and `reducer.ts`?

`export const SET_MODAL_DISMISSIBLE = 'SET_MODAL_DISMISSIBLE';`

Reply:
It depends on the code style. I followed the same style of all the other actions in this file, so probably I'd say it's fine for this PR, but you may want to discuss/open a separate issue to refactor the whole package.

TypeScript seems to be able to pick it up and validate against it anyway

Reply:
> TypeScript seems to be able to pick it up and validate against it anyway

Yup, this export in `actions.ts` is enough so we don't need to individually export/import all the action types: https://github.com/Automattic/wp-calypso/pull/47404/files#diff-f6abd693685fda89bf90415d7edcb77a7ee7af469f9b503369ae19ae9689cd73R129-R149